tests/expose_test.go

```diff
@@ -474,8 +797,9 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				Eventually(endptSubsetIpAddress).Should(BeEquivalentTo(vmPodIpAddress))
 
 				By("Deleting the VM.")
-				Expect(virtClient.VirtualMachine(vm.Namespace).Delete(vm.Name, &k8smetav1.DeleteOptions{})).To(Succeed())
-				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
+				Expect(cleanupVM(virtClient, vm.GetNamespace(), vm.GetName())).To(Succeed(), "should have been able to delete the VM object")
+				// this way, the test does not attempt to stop the VM in the `AfterEach` section
+				defer func() { vm = nil }()
```

Comment:
you can just `vm = nil` , without defer here, no need to add boilerplate since after this line it got to go directly